app/react/Layout/DocumentsList.js

```diff
@@ -39,26 +39,28 @@ class DocumentsList extends Component {
     }
   }
 
-  loadMoreDocuments(amount) {
+  loadMoreDocuments(amount, offset) {
```

Comment:
Should we take into account duplicates?

Example:
- You load library with 30 documents sorted by recently added
- Someone adds a new entity
- You select 'load more'

The last of the 30 documents will now be the first of the second batch, leaving you with a duplicate card.

Should we address that?

Reply:
confirmed this happens, and it gets really bad if someone published multiple documents at once, lets say 30, in that case the next batch load will be exactly the same documents as the first batch. 😂
how we can address that ? 
removing duplicated on client app will result on inconsistent load more numbers, you request for 30 more, 28 are loaded even though there is enough to load 30. which case do we prefer to have ? i to not have duplicated is better ? any other ideas @konzz @RafaPolit

Reply:
The team has decided not to address this for the time being.